### DISCUSSION TITLE
Conic Gradients for daily puzzle 14/12/2023

### DISCUSSION THREAD

**Discussion by sb1701e** [cssbattle/forum#373] [General]
When possible, I experiment using using backgrounds to accomplish parts or all of these challenges. When it comes to the puzzle from December 14, I though I had it made with 3 overlapping conic gradients, but sadly my efforts only get me to 99.9%. I've tweaked various parameters, and all I've done is lowered my score. What would I need to do to make this approach work, or is there a bug in the system that checks for sameness in this particular case?

`<p><style>body{background:#D4EFFC;margin:70 120}p{height:160;background:conic-gradient(#7BAFDE 25%,#000039 0)40px 40px/80px 80px no-repeat,conic-gradient(at 40px 80px,#D4EFFC 25%,#6867D4 0)40px 0/120px 120px no-repeat,conic-gradient(at 40px 40px,#0000 75%,#7BAFDE 0)0 0/120px 120px;rotate:-45deg`

**Comment by SelenIT** [comment]
I also haven't succeed with conic-gradient approach for that target. Probably playing with the order of layers can help (it helped with the box-shadow approach). Alternatively, sometimes blurring `body` a bit (like `filter:blur(.1q)`) helps to overcome the "99.9% hell" (again, helped me with box-shadow solution of this target, haven't checked it with gradients).
